components/org.wso2.carbon.identity.api.server.idv.provider/org.wso2.carbon.identity.api.server.idv.provider.common/src/main/java/org/wso2/carbon/identity/api/server/idv/provider/common/IdentityVerificationServiceHolder.java

```diff
@@ -18,14 +18,20 @@
 
 package org.wso2.carbon.identity.api.server.idv.provider.common;
 
+import org.wso2.carbon.context.PrivilegedCarbonContext;
 import org.wso2.carbon.extension.identity.verification.provider.IdVProviderManager;
 
 /**
  * Service holder class for identity verification Rest API.
  */
 public class IdentityVerificationServiceHolder {
 
-    private static IdVProviderManager idVProviderManager;
+    private IdentityVerificationServiceHolder() {};
+
+    private static class IdVProviderManagerHolder {
+        static final IdVProviderManager SERVICE = (IdVProviderManager) PrivilegedCarbonContext
```

Comment:
```suggestion

        static final IdVProviderManager SERVICE = (IdVProviderManager) PrivilegedCarbonContext
```

Reply:
will address this in a separate PR, as this needs to be fixed in all the components